test/integration/theme-schema.test.js

```diff
@@ -2,13 +2,18 @@
  * External dependencies
  */
 import Ajv from 'ajv-draft-04';
+import glob from 'fast-glob';
 
 /**
  * Internal dependencies
  */
 import themeSchema from '../../schemas/json/theme.json';
 
 describe( 'theme.json schema', () => {
+	const jsonFiles = glob.sync(
+		[ 'packages/*/src/**/theme.json', '{lib,phpunit,test}/**/theme.json' ],
+		{ onlyFiles: true }
+	);
```

Comment:
Similarly, we're now saying, any `theme.json` file anywhere within any of the packages should conform to the `https://schemas.wp.org/trunk/theme.json` schema. That also sounds good to me 👍

This is just thinking out loud, but I suppose there could theoretically be a use case for referring to a different schema, i.e. a relative path, or something? That seems pretty edge-casey to me, though, and if someone is attempting to do that, then hopefully the schema validation error here will signal to them what's going on, so I don't that's a blocker here.